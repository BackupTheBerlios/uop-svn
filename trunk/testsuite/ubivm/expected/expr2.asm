Entity start
	Constants
		3 I 1
		1 I 10
		2 I 20
		5 I 30
		6 I 5
		0 S start
		4 S writeln
	End
	Def start
		ldconst 1 --> [10]
		ldconst 2 --> [20]
		add
		ldconst 3 --> [1]
		lcall 4 --> [writeln]
		ldconst 1 --> [10]
		ldconst 2 --> [20]
		add
		ldconst 5 --> [30]
		add
		ldconst 3 --> [1]
		lcall 4 --> [writeln]
		ldconst 1 --> [10]
		ldconst 2 --> [20]
		add
		ldconst 6 --> [5]
		sub
		ldconst 3 --> [1]
		lcall 4 --> [writeln]
		stop
	End
End
