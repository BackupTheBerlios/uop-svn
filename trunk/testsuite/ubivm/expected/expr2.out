UbiVM - Release 0.1.0 (development release)
30
60
25
